internal/cmd/db_create.go

```diff
@@ -29,7 +29,7 @@ func init() {
 }
 
 var createCmd = &cobra.Command{
-	Use:               "create [flags] [database_name]",
+	Use:               "create [flags] [database-name]",
```

Comment:
Should we remove all occurrences of `[flags]`?

Reply:
I'm 50/50 on that. I like that we include it so it hints that there are optional flags, perhaps we move it after `[database-name]`.

I'm easy with whatever is decided.

Reply:
Almost all commands have flags, so I feel it is a bit redundant to add it in all the places.
It is also something that is easy to forget to add when there is a flag or forget to hide when there isn't any.